openslides/users/static/templates/users/user-import.html

```diff
@@ -68,7 +68,7 @@ <h3 translate>Select a CSV file
   <h4 translate>Please note:</h4>
   <ul>
       <li><translate>Required comma or semicolon separated values with these column header names in the first row</translate>:<br>
-          <code>title, first_name, last_name, structure_level, groups, comment, is_active, is_committee</code>
+          <code>title, first_name, last_name, structure_level, number, groups, comment, is_active, is_committee</code>
```

Comment:
You have to add the new number field also in the csv import preview table - see below...